landingzones/templates/landingzones/_zone_buttons.html

```diff
@@ -56,6 +56,14 @@
       Copy Zone UUID
     </a>
 
+    {% if zone.status != 'MOVED' and can_update_all or zone.user == request.user and can_update_own %}
+      <a class="dropdown-item sodar-lz-zone-btn-update"
+         href="{% url 'landingzones:update' landingzone=zone.sodar_uuid %}">
+        <i class="iconify" data-icon="mdi:update"></i>
```

Comment:
The convention is to use `mdi:lead-pencil` as the icon. I know, I should create that style guide some day.. :)